mappings/net/minecraft/item/ItemGroup.mapping

```diff
@@ -15,11 +23,32 @@ CLASS net/minecraft/unmapped/C_lnokcayd net/minecraft/item/ItemGroup
 		COMMENT <p>The name is rendered below the top row of item groups and above the inventory.
 	METHOD m_iwnppnlh setTexture (Ljava/lang/String;)Lnet/minecraft/unmapped/C_lnokcayd;
 		ARG 1 texture
+	METHOD m_mziozlja clearCache ()V
+	METHOD m_npmupqba provideStacks (Lnet/minecraft/unmapped/C_czxxrbcp;Lnet/minecraft/unmapped/C_lnokcayd$C_wnogvxsu;)V
+	METHOD m_nsgpxdxb getOrInitSearchStacks (Lnet/minecraft/unmapped/C_czxxrbcp;)Lnet/minecraft/unmapped/C_voakmxnc;
```

Comment:
`getOrCreateSearchStacks`?
```suggestion
	METHOD m_nsgpxdxb getOrCreateSearchStacks (Lnet/minecraft/unmapped/C_czxxrbcp;)Lnet/minecraft/unmapped/C_voakmxnc;
```

Reply:
The contribution doc says that `create` should be avoided for lazy initialization, so I searched for alternatives inside QM and got `init`

Reply:
Oh, my bad then